assets/xml/objects/object_bal.xml

```diff
@@ -1,81 +1,93 @@
+        <Texture Name="gTinglePenTopTex" OutName="tingle_pen_top" Format="rgba16" Width="16" Height="16" Offset="0x8AD0" />
+        <Texture Name="gTingleMapTex" OutName="tingle_map" Format="ci4" Width="64" Height="64" Offset="0x8CD0" />
+        <Texture Name="gTingleEyeClosedTex" OutName="tingle_eye_closed" Format="rgba16" Width="32" Height="64" Offset="0x94D0" />
+
+    <!-- Skeleton -->
+        <Limb Name="gTingleRootLimb" Type="Standard" EnumName="TINGLE_LIMB_ROOT" Offset="0xA4D0" />
+        <Limb Name="gTinglePelvisLimb" Type="Standard" EnumName="TINGLE_LIMB_PELVIS" Offset="0xA4DC" />
+        <Limb Name="gTingleBodyLimb" Type="Standard" EnumName="TINGLE_LIMB_BODY" Offset="0xA4E8" />
+        <Limb Name="gTingleUpperBodyLimb" Type="Standard" EnumName="TINGLE_LIMB_UPPER_BODY" Offset="0xA4F4" />
+        <Limb Name="gTingleBackRootLimb" Type="Standard" EnumName="TINGLE_LIMB_BACK_ROOT" Offset="0xA500" />
+        <Limb Name="gTingleBalloonLimb" Type="Standard" EnumName="TINGLE_LIMB_BALLOON" Offset="0xA50C" />
+        <Limb Name="gTingleRightArmRootLimb" Type="Standard" EnumName="TINGLE_LIMB_RIGHT_ARM_ROOT" Offset="0xA518" />
+        <Limb Name="gTingleRightShoulderLimb" Type="Standard" EnumName="TINGLE_LIMB_RIGHT_SHOULDER" Offset="0xA524" />
+        <Limb Name="gTingleRightUpperArmLimb" Type="Standard" EnumName="TINGLE_LIMB_RIGHT_UPPER_ARM" Offset="0xA530" />
+        <Limb Name="gTingleRightForearmLimb" Type="Standard" EnumName="TINGLE_LIMB_RIGHT_FOREARM" Offset="0xA53C" />
+        <Limb Name="gTingleRightWristLimb" Type="Standard" EnumName="TINGLE_LIMB_RIGHT_WRIST" Offset="0xA548" />
+        <Limb Name="gTingleRightHandLimb" Type="Standard" EnumName="TINGLE_LIMB_RIGHT_HAND" Offset="0xA554" />
+        <Limb Name="gTingleHeldPenLimb" Type="Standard" EnumName="TINGLE_LIMB_HELD_PEN" Offset="0xA560" />
+        <Limb Name="gTingleLeftArmRootLimb" Type="Standard" EnumName="TINGLE_LIMB_LEFT_ARM_ROOT" Offset="0xA56C" />
+        <Limb Name="gTingleLeftShoulderLimb" Type="Standard" EnumName="TINGLE_LIMB_LEFT_SHOULDER" Offset="0xA578" />
+        <Limb Name="gTingleLeftUpperArmLimb" Type="Standard" EnumName="TINGLE_LIMB_LEFT_UPPER_ARM" Offset="0xA584" />
+        <Limb Name="gTingleLeftForearmLimb" Type="Standard" EnumName="TINGLE_LIMB_LEFT_FOREARM" Offset="0xA590" />
+        <Limb Name="gTingleLeftWristLimb" Type="Standard" EnumName="TINGLE_LIMB_LEFT_WRIST" Offset="0xA59C" />
+        <Limb Name="gTingleLeftHandLimb" Type="Standard" EnumName="TINGLE_LIMB_LEFT_HAND" Offset="0xA5A8" />
+        <Limb Name="gTingleHeldMapLimb" Type="Standard" EnumName="TINGLE_LIMB_HELD_MAP" Offset="0xA5B4" />
+        <Limb Name="gTingleTorsoLimb" Type="Standard" EnumName="TINGLE_LIMB_TORSO" Offset="0xA5C0" />
+        <Limb Name="gTingleHeadLimb" Type="Standard" EnumName="TINGLE_LIMB_HEAD" Offset="0xA5CC" />
+        <Limb Name="gTingleRightHipLimb" Type="Standard" EnumName="TINGLE_LIMB_RIGHT_HIP" Offset="0xA5D8" /> 
```

Comment:
Similar to some of the above limbs, this is usually called the "right leg root" limb.